gpg/tasks/per_user.yml

```diff
@@ -0,0 +1,22 @@
+---
+- name: "Set vars for user {{ gpg_user_config.user }}"
+  set_fact:
+    gpg_basedir: "{{ gpg_user_config.gpg_basedir|default(gpg_default_basedir) }}"
```

Comment:
I can't see `gpg_user_config.gpg_basedir` mentioned anywhere - should this be in the example config?

Reply:
See here: https://github.com/TomAdam/ansible-roles/pull/18/files#diff-ec27784dcffa210deef91f89332bf09cR15

Loop var allows you to alias a var within the loop which allows you to have loops inside loops. Also makes it simpler. Effectively it would be an item in `gpg_config`.

Reply:
Yeh I got that, but you haven't mentioned `gpg_basedir` anywhere. I can see it defaults to the default, but maybe it would be clearer to add it into the readme example config too?

Reply:
Good point, didn't read your comment properly. I'll push up some more notes. Going forward I want to start adding readmes to the roles that are more complicated. I always spend ages figuring out how to configure them.